staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -315,9 +317,19 @@ func cleanVerb(verb string, request *http.Request) string {
 	if verb == "WATCHLIST" {
 		reportedVerb = "WATCH"
 	}
+	if verb == "PATCH" && request.Header.Get("Content-Type") == string(types.ApplyPatchType) {
+		reportedVerb = "APPLY"
+	}
 	return reportedVerb
 }
 
+func cleanDryRun(dryRun []string) string {
+	if len(dryRun) != 0 {
```

Comment:
Since this is supposed to be a list of string, and we currently only support one string, maybe you should keep it as the input string?